src/main/resources/schemas/policy-schema-form.json

```diff
@@ -1,107 +1,137 @@
 {
-  "type" : "object",
-  "id" : "urn:jsonschema:io:gravitee:policy:groovy:configuration:GroovyPolicyConfiguration",
-  "properties" : {
-    "scope" : {
-      "title": "Scope",
-      "type" : "string",
-      "default": "REQUEST",
-      "enum" : [ "REQUEST", "RESPONSE"],
-      "x-schema-form": {
-        "hidden": true
-      }
-    },
-    "onRequestScript" : {
-      "title": "On-request script",
-      "description": "Groovy script to evaluate during the OnRequest phase.",
-      "type" : "string",
-      "x-schema-form": {
-        "type": "codemirror",
-        "codemirrorOptions": {
-          "placeholder": "Put your groovy script here or drag'n'drop groovy script",
-          "lineWrapping": true,
-          "lineNumbers": true,
-          "allowDropFileTypes": true,
-          "autoCloseTags": true,
-          "mode": "groovy"
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Something comes to mind: do these schema changes are compatible with AM (UI / Gateway)?

Reply:
@jhaeyaert ok I just checked on am master and as it works in apim with the old policy studio, I can't see any reason it would not work as they are using it as well.

Reply:
all good, tested on master, the policy is usable on the UI and the execution is on on GW side